manifests/init.pp

```diff
@@ -240,6 +240,31 @@
 #   The RabbitMQ port.
 # @param python_package
 #   Name of the package required by rabbitmqadmin.
+# @param quorum_membership_reconciliation_enabled
+#   Boolean - Enables or disables continuous membership reconciliation.
+#   This requires RabbitMQ 3.13 or higher. More information on this configuration
+#   can be found here: https://www.rabbitmq.com/docs/quorum-queues
+# @param quorum_membership_reconciliation_auto_remove
+#   Boolean - Enables or disables automatic removal of member nodes that are no longer part of the cluster, 
+#   but still a member of the quorum queue. This requires RabbitMQ 3.13 or higher. More information on this configuration
```

Comment:
Maybe don't need to have the version requirement and links to docs in each individual parameter, just the enabled one, and then mention for the others that the top level parameter needs to be enabled?

Reply:
That makes sense - done!